src/util/PermalinkUtil/PermalinkUtil.ts

```diff
@@ -0,0 +1,64 @@
+import OlMap from 'ol/Map';
+import OlLayer from 'ol/layer/Base';
+import TileLayer from 'ol/layer/Tile';
+import MapUtil from '@terrestris/ol-util/dist/MapUtil/MapUtil';
+
+/**
+ * Helper class for some operations related to permalink function.
+ *
+ * @class
+ */
+export class PermalinkUtil {
```

Comment:
This one is the great candidate for the ol-util

Reply:
https://github.com/terrestris/ol-util/pull/401

Reply:
❤️